torch/_functorch/_aot_autograd/schemas.py

```diff
@@ -106,7 +106,7 @@ class MutationType(Enum):
 
 
 # This class tells us info about user inputs.
-@dataclass(frozen=True)
```

Comment:
Could we keep this frozen and use `dataclasses.replace` to add these values instead? Unless the copy is going to be too much of a perf hit, the benefit of keeping the data structure immutable will be really helpful. Most pre compile wrappers create a new copy of ViewAndMutationData anyway.

Reply:
Thanks, I did not know about dataclasses.replace - applied.

Reply:
Moved fields from InputAliasInfo to ViewMutationsMeta field, keeping InputAliasInfo immutable